src/index.html

```diff
@@ -6,8 +6,285 @@
+        <div class="top">
+          <div class="title">
+            <h2>EDUCATION</h2>
+          </div>
+          <div class="media">
+            <a href="">DOWNLOAD MY FULL RESUME</a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
+            <div class="education">
+              <h3>The San Francisco Business School</h3>
+              <h4>2010-2012</h4>
+              <p>
+                This is a description of your education. Concisely describe your
+                certifiate, degree or course in a way that’s easy for readers to
+                scan quickly.
+              </p>
+            </div>
+            <div class="education">
+              <h3>NYC School Of Marketing and Advertising</h3>
+              <h4>2007-2010</h4>
+              <p>
+                This is a description of your education. Concisely describe your
+                certifiate, degree or course in a way that’s easy for readers to
+                scan quickly.
+              </p>
+            </div>
+            <div class="education">
+              <h3>L.A. University of Applied Science</h3>
+              <h4>2002-2006</h4>
+              <p>
+                This is a description of your education. Concisely describe your
+                certifiate, degree or course in a way that’s easy for readers to
+                scan quickly.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Hobbies & Interests Slider-->
+      <div class="slider">
+        <div class="top">
+          <div class="title">
+            <h2>HOBBIES & INTERESTS</h2>
+          </div>
+          <div class="media">
+            <a href="">DOWNLOAD MY FULL RESUME</a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-volleyball-ball"></i>
+              <p>Sports</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-theater-masks"></i>
+              <p>Thratre</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-utensils"></i>
+              <p>BBQ</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-glass-cheers"></i>
+              <p>Party</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-cookie"></i>
+              <p>Cooking</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Skill Section-->
+    <div class="skills">
+      <div class="skill">
+        <p class="skill_name">Microsoft Excel</p>
```

Comment:
It's better to follow the class/id naming convention: kebab-case or snake_case:
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841